python/pickle/PyROOT_readtests.py

```diff
@@ -107,11 +107,15 @@ def __doftest( self, d ):
    def test5ReadCustomTypes( self ):
       """Test reading PyROOT custom types"""
 
+      proto = [ctypes.c_long(123), ctypes.c_double(123.123)]
+
       o = pickle.load( self.in1 )
-      self.assertEqual(o, [123,123.123] )
+      for touple in zip(o, proto):
```

Comment:
Instead of `tuple` you can put here `for e1,e2 in zip(o, proto)` so that Python already binds the list elements to two variables and you do not need to index the tuple afterwards.